src/clusterfuzz/_internal/system/environment.py

```diff
@@ -1121,6 +1122,11 @@ def is_android_real_device():
   return base_platform(platform()) == 'ANDROID'
 
 
+def is_using_mte():
+  """Return True if device is using Memory Tagging Extension."""
+  return "USES_MTE" in os.environ
```

Comment:
In CF, the convention is to do environment.get_value('USE_MTE'). That way USE_MTE=0 doesn't make this function return True. The odd behavior of our approach though is it treats the values as python code. So USE_MTE=False, is `False` unlike in most programs where it would be considered True since it is non-zero.

Reply:
Ack.

Updated.